includes/Customizer.php

```diff
@@ -0,0 +1,600 @@
+		$value = intval( $value );
+
+		if ( $value <= 0 ) {
+			$validity->add( 'invalid_number', __( 'The number of stories must be between 1 and 4.', 'web-stories' ) );
+		} elseif ( $value > 5 ) {
+			$validity->add( 'invalid_number', __( 'The number of stories must be between 1 and 4.', 'web-stories' ) );
+		}
+		return $validity;
+	}
+
+	/**
+	 * Renders web stories based on the customizer selected options.
+	 *
+	 * @SuppressWarnings(PHPMD.NPathComplexity)
+	 * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+	 *
+	 * @return string|void
+	 */
+	public static function render_stories() {
+		$options = get_option( 'story-options' );
+
+		if ( empty( $options['show_stories'] ) || true !== $options['show_stories'] ) {
+			return;
+		}
+
+		$theme_support = get_theme_support( 'web-story-options' );
+		$theme_support = ! empty( $theme_support[0] ) && is_array( $theme_support[0] ) ? $theme_support[0] : [];
+
+		$default_array = [
+			'view_type'             => ( ! empty( $theme_support['view-type-default'] ) ) ? $theme_support['view-type-default'] : 'circles',
+			'show_title'            => ( ! empty( $theme_support['title-default'] ) ) ? $theme_support['title-default'] : false,
+			'show_author'           => ( ! empty( $theme_support['author-default'] ) ) ? $theme_support['author-default'] : false,
+			'show_date'             => ( ! empty( $theme_support['date-default'] ) ) ? $theme_support['date-default'] : false,
+			'stories_archive_label' => ( ! empty( $theme_support['stories-archive-label'] ) ) ? $theme_support['stories-archive-label'] : __( 'View all stories', 'web-stories' ),
+			'show_story_poster'     => ( ! empty( $theme_support['show-story-poster-default'] ) ) ? $theme_support['show-story-poster-default'] : true,
+			'number_of_columns'     => ( ! empty( $theme_support['grid-columns-default'] ) && is_numeric( $theme_support['grid-columns-default'] ) ) ? $theme_support['grid-columns-default'] : 2,
+		];
+
+		$query_arguments = [
+			'posts_per_page' => ( ! empty( $theme_support['number-of-stories'] ) && is_numeric( $theme_support['number-of-stories'] ) ) ? $theme_support['number-of-stories'] : 5,
+		];
+
+		$query_arguments['posts_per_page'] = ! empty( $options['number_of_stories'] ) ? $options['number_of_stories'] : $query_arguments['posts_per_page'];
+
+		$order_by = ! empty( $theme_support['order-default'] ) ? $theme_support['order-default'] : 'latest';
+		$order_by = ! empty( $options['order'] ) ? $options['order'] : $order_by;
+
+		switch ( $order_by ) {
+			case 'oldest':
+				$query_arguments['order'] = 'ASC';
+				break;
+			case 'alphabetical':
+				$query_arguments['orderby'] = 'title';
+				$query_arguments['order']   = 'ASC';
+				break;
+			case 'reverse-alphabetical':
+				$query_arguments['orderby'] = 'title';
+				$query_arguments['order']   = 'DESC';
+				break;
+			case 'random':
+				$query_arguments['orderby'] = 'rand'; //phpcs:ignore WordPressVIPMinimum.Performance.OrderByRand.orderby_orderby
```

Comment:
We can create a wrapper function same as VIP's to avoid this phpcs error. Function will have same name as VIP's function, but there would be a `function_exists` check on top of it.

Reply:
This would be handled inside the `Story_Query` class right?